### PR TITLE
Rachel/user deletion

### PR DESCRIPTION
**Pull request by blakerach1**
## Describe your changes

- set up API's to GET User and GET Users and DELETE User
- set up hook's for use-user and use-users
- set up UserCard component
- set up UserPage page and UsersPage
- UsersPage will only return info when admin user is logged in. 

Work to be done on getting the error handling for when a non admin user visits the url... can't get the html to render if there is an caught at the API fetch stage...

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## What kind of feedback do you need?

Quick look please.

## Checklist:

- [x] I have performed a self-review of my code
- [x] I have recommended where to start and how to proceed with the review - **follow in the order of changes described**

### CODE REVIEW

**Review by shayzimm — APPROVED** [review]

File: src/api/delete-user.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+async function deleteUser(username) {
+    // const url = `${import.meta.env.VITE_API_URL}/users/${username}`;
```
Comment (Lyndallh) [inline src/api/delete-user.js]: pointing at local (just reminded me to change in my upcoming PR!)

File: src/api/get-users.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+async function getUsers(){
+    // const url=`${import.meta.env.VITE_API_URL}/users`;
+    // to test in local: comment line above and uncomment line below (also check url in line below matches your local backend url)
+    const url=`http://127.0.0.1:8000/users`;
```
Comment (Lyndallh) [inline src/api/get-users.js]: same again ;-)

**Review by Lyndallh — CHANGES_REQUESTED** [review]

**Review by ktechgau — APPROVED** [review]

**Review by shayzimm — APPROVED** [review]

File: src/api/delete-user.js (hunk 1 above)
Comment (blakerach1) [reply src/api/delete-user.js]: Fixed :)

File: src/api/get-users.js (hunk 2 above)
Comment (blakerach1) [reply src/api/get-users.js]: fixed :)

**Review by Lyndallh — APPROVED** [review]

**Pull request merged**